### PR TITLE
Add org.kde.kmahjongg

### PR DESCRIPTION
**Pull request by cho2**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of [permissions](https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.kmahjongg.json#L10). 
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
  - I am a member of [KDE Team on Flathub](https://github.com/orgs/flathub/teams/kde/members).
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - KDE project owns the domain
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
  - n/a

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by cho2** [comment]
bot, build org.kde.kmahjongg

File: org.kde.kmahjongg.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kmahjongg",
+    "rename-icon": "kmahjongg",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=fallback-x11",
```
Comment (JakobDev) [inline org.kde.kmahjongg.json:12]:
```suggestion
        "--socket=wayland",
        "--socket=fallback-x11",
```

`--socket=fallback-x11` is enough

**Comment by cho2** [comment]
cc @flathub/kde

**Comment by travier** [comment]
Manifest LGTM with the same comment as above. Not tested yet.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.kmahjongg.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+    "id": "org.kde.kmahjongg",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
```
[the remark this answers is not in the document]
Comment (JakobDev) [reply org.kde.kmahjongg.json:4]: The KDE team wants to use 5.15-21.08 at the moment. [Source](https://github.com/flathub/org.kde.okular/pull/197#issuecomment-1240694420)

[the remark this answers is not in the document]
Comment (travier) [reply org.kde.kmahjongg.json:4]: Yes, please do not use the runtime before we've announced it

[the remark this answers is not in the document]
Comment (travier) [reply org.kde.kmahjongg.json:4]: I'm trying to test it a bit more before we announce it and convert all apps to it

**Comment by tsdgeos** [comment]
App seems to work fine

**Pull request closed without merging**